extensions/amp-story/1.0/animation.js

```diff
@@ -248,11 +290,19 @@ export class AnimationRunner {
       this.runner_.cancel();
     }
 
-    return this.firstFrameProps_.then((firstFrameProps) =>
-      this.vsync_.mutatePromise(() => {
-        setStyles(this.target_, assertDoesNotContainDisplay(firstFrameProps));
-      })
-    );
+    return this.firstFrameProps_.then((firstFrameProps) => {
+      if (!firstFrameProps) {
+        // These are only available when they can be evaluated. They can't
```

Comment:
Can you add a few words to say when they can be evaluated? I know you already wrote it earlier in this PR but it looks useful to repeat here.